conans/test/conftest.py

```diff
@@ -34,6 +34,7 @@
 }
 """
 
+MacOS_M1 = all([platform.system() == "Darwin", platform.machine() == "arm64"])
```

Comment:
mmm if one of my two branches for fixing the tests on Apple Silicon (I had a few variations...) I had something along the lines of:

```
homebrew_root = "/opt/homebrew" if MacOS_arm else "/usr/local"
```

then you can reference this on the macos paths and avoid more conditionals.